### DISCUSSION TITLE
Wiki.js Documentation Site Broken

### DISCUSSION THREAD

**Discussion by Pingu-Pox** [requarks/wiki#4033] [Error / Bug Report] [answered]
Trying to use the official documentation page, but getting the following errors:

Docker Flow Proxy: 503 Service Unavailable
No server is available to handle this request.

To be clear, this is a problem with the official wiki.js documentation webpage, not anything I am running.

Anyone else having issues?

EDIT: To be clear, in the docs page a reproduceable issue I am having on two machines on different networks is that I cannot access "Manage Pages" under "User Guide" of the docs.

**Comment by engineereng** [comment]
I'm getting the exact same issue when I access the documentation at https://docs.requarks.io/. The 1.0 documentation site at https://docs-legacy.requarks.io/ seems to be working, though.

EDIT: After some fiddling around, it seems that the 503 error appears occasionally. Refreshing the page worked.

**Reply by Pingu-Pox** [reply]
Yeah I noticed if you keep refreshing you can get the page sometimes. Hope the wiki team is aware of it.

**Comment by AjdinDev** [comment]
@NGPixel This issue is still present.

**Comment by NGPixel** [comment, marked as answer]
The issue has been resolved.
